Recruit-Training/Basic-Recruit-Training/person/Person.cpp

```diff
@@ -0,0 +1,44 @@
+#include "Person.h"
+#include <iostream>
+
+Person::Person()
+: age_(new int[1]) {}
+
+Person::~Person(){
+	delete age_;
+}
+
+Person::Person(const std::string& name, int age) 
+: name_(name), age_(new int(age)) {}
+
+int Person::getAge() const {
+	return age_[0];
```

Comment:
When you update something please also check where it's being use in other places.